cookiecutter.json

```diff
@@ -6,6 +6,11 @@
     "monitoring": "y",
     "monitor_view_execution_time_in_djagno": "y",
     "monitor_tcpstat": "y",
+    "log_aggregating": "y",
+    "log_aggregator_url": "https://loki.reef.pl",
+    "log_aggregator_user": "acme",
```

Comment:
acme has some meaning? IMO best if we leave USER out of cookiecutter templating vars as only use it sees is another template and it HAS to be changed

Reply:
I don't understand. It must be properly set by user. "Acme" is just a dummy value (popular word for fake company), the same as "project" for repo name is something that must be changed.

Reply:
doesn't sound like it bring any value here, so lets throw it out from here and have just empty `LOKI_USER=` in `.env.template`, just like `LOKI_PASSWORD=` and any other prod-only credential

Reply:
Fine for me, but I need to remove `log_aggregator_client` and `log_aggregator_client_server_group` as well